app/src/main/res/layout/item_reservation.xml

```diff
@@ -27,8 +32,7 @@
             android:id="@+id/text_hour_start"
             android:layout_width="120dp"
             android:layout_height="wrap_content"
-
-
+            android:textAlignment="center"
             android:layout_marginBottom="16dp"
             android:text="TextView"
```

Comment:
Do not let hardcode text in the xml, always try to fetch from the resource/values/string folder

Reply:
ok, do you mind if I give a moderate priority to this part?